x-pack/plugin/sql/qa/src/main/resources/agg.csv-spec

```diff
@@ -137,3 +137,80 @@ null           |null           |null           |null           |null           |
 3              |3              |51             |3              |3.0            |100.0          |NaN            |NaN            
 4              |4              |72             |4              |4.0            |0.0            |NaN            |NaN   
 ;
+
+
+//
+// Grouping functions
+//
+
+
+histogramNumeric
+SELECT HISTOGRAM(salary, 5000) AS h FROM test_emp GROUP BY h;
```

Comment:
don't we need ORDER BY to avoid flakiness?

Reply:
No since all groups have an implicit order (due to how composite agg works).